nltk/tag/stanford.py

```diff
@@ -204,6 +205,79 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, url='http://localhost:9000', encoding='utf8'):
```

Comment:
I favour merging with the current, restricted interface. A possible enhancement is to filter *args and *kwargs, only removing arguments that could be used to change the semantics in undesirable ways.